src/bin/pg_autoctl/keeper_pg_init.c

```diff
@@ -709,6 +709,15 @@ create_database_and_extension(Keeper *keeper)
 	/* publish our new pgSetup to the caller postgres state too */
 	postgres->postgresSetup.ssl = initPostgres.postgresSetup.ssl;
 
+	/* Ensure pg_stat_statements is installed in the server extension dir used
+	 * to create the Postgres instance
+	 */
```

Comment:
Minor style issue: the first line of the comments is always empty in the code base, for reasons...

Reply:
Whoops, sorry I missed that